kubernetes-azure-go/main.go

```diff
@@ -0,0 +1,171 @@
+package main
+
+import (
+	"encoding/base64"
+
+	"github.com/pulumi/pulumi-azure-native/sdk/go/azure/containerservice"
+	"github.com/pulumi/pulumi-azure-native/sdk/go/azure/network"
+	"github.com/pulumi/pulumi-azure-native/sdk/go/azure/resources"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi/config"
+)
+
+func main() {
+	pulumi.Run(func(ctx *pulumi.Context) error {
+		// Get some configuration values or set default values
+		cfg := config.New(ctx, "")
+		sshPubKey := cfg.Require("sshPubKey")
+		mgmtGroup := cfg.Require("mgmtGroupId")
+		prefixForDns, err := cfg.Try("prefixForDns")
+		if err != nil {
+			prefixForDns = "pulumi"
+		}
+		kubernetesVersion, err := cfg.Try("kubernetesVersion")
+		if err != nil {
+			kubernetesVersion = "1.24.3"
+		}
+		numWorkerNodes, err := cfg.TryInt("numWorkerNodes")
+		if err != nil {
+			numWorkerNodes = 3
+		}
+		nodeVmSize, err := cfg.Try("nodeVmSize")
+		if err != nil {
+			nodeVmSize = "Standard_DS2_v2"
+		}
+
+		// Create an Azure Resource Group
+		resourceGroup, err := resources.NewResourceGroup(ctx, "resourceGroup")
```

Comment:
```suggestion
		resourceGroup, err := resources.NewResourceGroup(ctx, "resourceGroup", &resources.ResourceGroupArgs{})
```

Reply:
Yep, this caused an error. Fixed in 8cb24a3.